### PR TITLE
fix(engines): add nft annotation for libquery_engine

### PR DESCRIPTION
**Pull request by williamluke4**

### CODE REVIEW

File: packages/engines/src/index.ts [hunk 1]
```diff
@@ -1,32 +1,32 @@
-import path from 'path'
-import { enginesVersion } from '@prisma/engines-version'
-import { download } from '@prisma/fetch-engine'
+import { enginesVersion } from "@prisma/engines-version";
+import { download } from "@prisma/fetch-engine";
+import path from "path";
```
Comment (Jolg42) [inline packages/engines/src/index.ts:3]: Not sure why the double quotes and ; made it here.

Would be good to have the same setting as here https://github.com/prisma/prisma/blob/master/src/.prettierrc.yml

**Comment by Jolg42** [comment]
Should we merge this today?

File: packages/engines/src/index.ts (hunk 1 above)
Comment (williamluke4) [reply packages/engines/src/index.ts:3]: Yeah prettier just needs to be setup. I Will add that in this PR

**Review by Jolg42 — APPROVED** [review]
Thanks for adding Prettier ❤️

**Pull request merged**